teams/templates/includes/create_slack_mpim.html

```diff
@@ -0,0 +1,16 @@
+{% if create_group_im and request.user in team.participants.all %}
+{% if team.communication_channel %}
+<a role="button" class="btn btn-sm btn-ci" href="{{team.communication_channel}}" taget="_blank">
+    <i class="fab fa-slack fa-lg ml-0 mr-2 mt-1 mb-1 p-orange"></i>
+    Go To Slack Group
+</a>
+{% else %}
+<form action="/teams/{{team.id}}/create_group_im/" method="POST">
+{% csrf_token %}
+<button class="btn btn-sm btn-ci">
+    <i class="fab fa-slack fa-lg ml-0 mr-2 mt-1 mb-1 p-orange"></i>
+    Create Group IM in Slack
+</button>
+</form>
+{% endif %}
+{% endif %}
```

Comment:
Missing new line at end of file.
Also, a bit of better formatting (indentation) for these multiple `{% if %}` statements would be ideal.

Reply:
Updated as suggested and added indentation.